_data/team-academic.yml

```
@@ -73,25 +76,80 @@
   thumbnailUrl: "Moritz Lennert.jpg"
 - name: "Marius"
   surname: "Budileanu"
+  title: "Faculty of Geography, University of Bucharest"
   thumbnailUrl: "Marius Budileanu.jpg"
 - name: "Tuong-Thuy"
   surname: "Vu"
   thumbnailUrl: "Tuong-Thuy Vu.jpg"
 - name: "Dietrich"
   surname: "Schröder"
+  title: "Faculty of Geomatics, Computer Science and Mathematics, Hochschule für Technik Stuttgart"
   thumbnailUrl: "Dietrich Schröder.jpg"
 - name: "Franz-Josef"
   surname: "Behr"
+  title: "Stuttgart University of Applied Sciences"
   thumbnailUrl: "Franz-Josef Behr.png"
 - name: "Bolelang"
   surname: "Sibolla"
   title: "Council for Scientific and Industrial Research"
-  thumbnailUrl: "generic.jpg"
+  thumbnailUrl: "Bolelang_Sibolla.jpg"
 - name: "Codrina"
   surname: "Maria Ilie"
   title: "Terrasigna"
-  thumbnailUrl: "generic.jpg"
+  thumbnailUrl: "Codrina_Ilie.jpg"
 - name: "Sisi"
   surname: "Zlatanova"
   title: "University of New South Wales"
+  thumbnailUrl: "Sisi_Zlatanova.jpg"
+- name: "Peter"
+  surname: "Mooney"
+  title: "Maynooth University"
+  thumbnailUrl: "Peter_Mooney.png"
```

Please keep image name consistent: `Firstname Lastname.extension`
```suggestion
  thumbnailUrl: "Peter Mooney.png"
```